<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>911</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lName">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtName">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);
color: rgb(170, 0, 0);
margin-bottom:8px;
padding:10px;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Enter Name</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lEmail">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtEmail">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);
color: rgb(170, 0, 0);
margin-bottom:8px;
padding:10px;</string>
      </property>
      <property name="placeholderText">
       <string>Enter Email</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lMobileno">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Phone no:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtMobileno">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);
color: rgb(170, 0, 0);
margin-bottom:8px;
padding:10px;</string>
      </property>
      <property name="placeholderText">
       <string>Enter MobileNo</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lGender">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Gender:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="txtCombobox">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);
color: rgb(170, 0, 0);
margin-bottom:8px;
padding:10px;</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>--Select Gender--</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Others</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lDob">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Dob:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateEdit" name="txtDate">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);
color: rgb(170, 0, 0);
margin-bottom:8px;
padding:10px;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lAddress">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);</string>
      </property>
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="txtAddress">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
font: 79 4pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);
color: rgb(170, 0, 0);
margin-bottom:4px;
</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Black'; font-size:4pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 133, 180);
background-color: rgb(255, 255, 255);
border-radius:10px;
color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 133, 180);
background-color: rgb(255, 255, 255);
border-radius:10px;
color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 133, 180);
background-color: rgb(255, 255, 255);
border-radius:10px;
color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>490</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 133, 180);
background-color: rgb(255, 255, 255);
border-radius:10px;
color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>601</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 133, 180);
background-color: rgb(0, 0, 0);

border-radius:10px;
color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);
</string>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phone No</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gender</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dob</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Address</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="Empmngsys">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 133, 180);
background-color: rgb(0, 0, 127);

border-radius:10px;
font:24pt;
font-family:sans-serif;
font: 28pt &quot;Algerian&quot;;
border:6px groove white;
</string>
   </property>
   <property name="text">
    <string>Emp Management System</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowtable">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>710</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 133, 180);
background-color: rgb(255, 255, 255);
border-radius:10px;
color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Show Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>911</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>krishn.jpg.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>btnAdd</zorder>
  <zorder>btnUpdate</zorder>
  <zorder>btnDelete</zorder>
  <zorder>btnSearch</zorder>
  <zorder>tableWidget</zorder>
  <zorder>Empmngsys</zorder>
  <zorder>btnShowtable</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
